sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -1695,7 +1698,7 @@ public void validate(PCollection<TableRow> input) {
             "CreateDisposition is CREATE_IF_NEEDED, however no schema was provided.");
 
         // The user specified a table.
-        if (jsonTableRef != null && validate) {
+        if (jsonTableRef != null) {
```

Comment:
If I read correctly, sounds like this is a new bug? I'll summarize what I think you're saying. If so, please file a JIRA issue.

* `BigQueryIO.Write.to("some-table-that-exists")` does not require a schema, as long as that table exists.
* `.withoutValidation` disables checking that the table exists, for a variety of reasons.

These two properties come into conflict: it ought to be legal to combine them, but instead validation fails (with what error?). Options:

1. Should we change the semantics here so that the user has to set CREATE_NEVER if not supplying a schema? Then ensure this scenario works with and without validation.

2. If not, we should probably log a warning that validation is disabled, but no schema was supplied, and as a result this pipeline may fail when writing data.